tests/unit_tests/analysis/test_es_update.py

```diff
@@ -139,6 +140,23 @@ def test_update_report(snake_oil_case_storage, snake_oil_storage, snapshot):
                 -1.2135225153906364,
                 1.27516244886867,
             ],
+            False,
+        ),
+        (
+            "STD_ENKF",
+            [
+                0.4658755223614102,
+                0.08294244626646294,
+                -1.2728836885070545,
+                -0.7044037773899394,
+                0.0701040026601418,
+                0.25463877762608783,
+                -1.7638615728377676,
+                1.0900234695729822,
+                -1.2135225153906364,
+                1.27516244886867,
```

Comment:
Seeing as the goal of the test is to make sure that these values are the same as the ones above, perhaps we should split them out in a variable, so they can accidentally drift apart?

Reply:
Something like this: https://github.com/equinor/ert/pull/6427/commits/3ecf6866cff33bc389f27b9d41694e37ebb5d80a
?